hack/test-e2e.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+#shellcheck disable=SC2086
+
+set -e
+
+image="origin-libvirt-machine-controllers:latest"
+script_dir="$(cd $(dirname "${BASH_SOURCE[0]}") && pwd -P)"
+
+# Set packet instance connection parameters
+RHOST="root@$(cat /tmp/packet_ip)"
+SSH_OPTS="-o UserKnownHostsFile=/dev/null -o IdentitiesOnly=yes -o PasswordAuthentication=no -o StrictHostKeyChecking=no -i /tmp/packet_id_rsa"
+
+# TODO remove after https://github.com/paulfantom/packet-image/pull/1
+ssh ${SSH_OPTS} "${RHOST}" 'yum -y -d1 install docker && systemctl start docker'
```

Comment:
In case the guest reboots, the docker will by down. `&& systemctl enable docker &&` ?